website/src/hugo/content/getting-help.md

```diff
@@ -2,14 +2,13 @@
 # Getting Help
 
 
-## Gitter
+## Discord
 
-The discussion forum for http4s is [http4s/http4s on Gitter][Gitter].
-This channel is open to user questions of all levels of experience.
-More general questions about Cats, FS2, HTTP, Scala, etc. are always
-welcome, but may politely be redirected to a more specific forum.  We
-are less concerned about staying strictly on topic than helping
-people.  If in doubt, ask.
+The discussion forum for http4s is [http4s/http4s on the Typelevel
+Discord][Discord].  This Discord is open to user questions of all
+levels of experience.  There you will also find channels for questions
+and discussion about Cats, FS2, skunk, and much more.  Please try to
+keep your questions and contributions in these channels topical.
```

Comment:
This feels mildly unwelcoming to me - is it addressing a real problem?

Reply:
That's a good point, let's just drop it?
```suggestion
and discussion about Cats, FS2, skunk, and much more!
```

Reply:
If we want to have a sentence, maybe it should be about Scala CoC and everyone is welcome :)